src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -275,9 +299,11 @@ class MockClientStats {
   }
 
  private:
-  std::vector<std::chrono::milliseconds> response_delays{
+  std::vector<std::chrono::milliseconds> response_delays_{
       std::chrono::milliseconds{0}};
-  std::atomic<size_t> response_delays_index{0};
+  std::vector<cb::Error> response_statuses_{cb::Error::Success};
```

Comment:
Why are this and `response_delays_` pre-populated?

Reply:
They need a default for the default case where no one specifies any delays or statuses. The default is 0 delay, and all requests return success.